### PR TITLE
awslambda: add single and strip_path_prefix docs

### PR DESCRIPTION
**Pull request by coopernurse**
Add docs for `single` and `strip_path_prefix` config options.

### CODE REVIEW

File: content/docs/awslambda.md [hunk 1]
```diff
@@ -36,16 +38,21 @@ the Lambda functions to conform to the response JSON envelope format.
...
+
+For example, given a directive `awslambda /lambda/`, requests to `/lambda/hello-world` and `/lambda/hello-world/abc`
+would each invoke the AWS Lambda function named `hello-world`.
 
 The `include` and `exclude` globs are simple wildcards, not regular expressions.
 For example, `include foo*` would match `food` and `footer` but not `buffoon`, while
 `include *foo*` would match all three.
 
-`include` and `exclude` rules are run before `name_prepend` and `name_append` are applied.
+`include and `exclude` rules are run before `name_prepend` and `name_append` are applied and
```
Comment (mholt) [inline content/docs/awslambda.md]: Is this line formatted right? GitHub's markdown coloring is being thrown off... extra backtick maybe?

File: content/docs/awslambda.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ the Lambda functions to conform to the response JSON envelope format.
     <span class="hl-subdirective">exclude</span>       <i>excluded function names...</i>
     <span class="hl-subdirective">name_prepend</span>  <i>string to prepend to function name</i>
     <span class="hl-subdirective">name_append</span>   <i>string to append to function name</i>
+    <span class="hl-subdirective">single</span>   <i>name of a single lambda function to invoke</i>
+    <span class="hl-subdirective">strip_path_prefix</span>   <i>If true, path and function name are stripped from the path</i>
```
Comment (mholt) [inline content/docs/awslambda.md]: Are these aligned (in two columns)?

**Comment by coopernurse** [comment]
Thanks for the feedback - alignment of syntax table and missing backtick have been fixed.

**Comment by mholt** [comment]
Thanks!

**Pull request merged**